app/api/events.py

```diff
@@ -51,13 +51,14 @@
 from app.models.stripe_authorization import StripeAuthorization
 
 
-def check_ticketing(user, modules):
+def check_ticketing(user, modules, data=None):
     if not user.is_verified:
         raise ForbiddenException({'source': '/data/attributes/is-verified'},
                                  "Please verify your Email")
-    elif not modules.ticket_include:
-        raise ForbiddenException({'source': '/data/attributes/ticket-include'},
-                                 "Ticket modules not included.")
+    else:
+        if data.get('is_ticketing_enabled', False) and not modules.ticket_include:
```

Comment:
The condition is wrong, it should check `data.get('is_ticketing_enabled', False)`

Please check logic before pushing